docker-compose.yml

```diff
@@ -13,10 +13,19 @@ services:
       MYSQL_USER: takwimu
       MYSQL_PASSWORD: takwimu
       MYSQL_ROOT_PASSWORD: takwimu
+  es:
+    build: ./docker/elasticsearch
+    ports:
+      - "9200:9200"
+    environment:
+      discovery.type: single-node
+      ES_JAVA_OPTS: -Xms512m -Xmx512m
```

Comment:
Firefox complains about CORS. We need to enable it here:
```
      http.cors.enabled: "true"
      http.cors.allow-origin: "*"
```
Or something like that.

Reply:
:+1: